examples/Flowchart.elm

```diff
@@ -145,97 +168,130 @@ arrow length =
 
 render : Flow -> Collage msg
 render flow =
```

Comment:
I know it's my own fault, that I used a custom formatter, but I take you didn't change much or anything at all in this function?

Reply:
Oops, yeah sorry about all the formatting changes, accidentally ran elm-format on this one. I did just change one thing in this function - I made the connecting line stop at the sides of the diamond instead of running behind it where it's visible behind the sketched fill.

Reply:
Sounds good.

The formatting thing is actually my fault, as I used a custom formatter. (I don't like four space indentations and find all the newlines between case branches way to much noise...) As elm-format is the community standard, I could run it on all source files and make a commit on master, you can do the same and rebase on that? Or would that give us a lot of whitespace errors?